tests/src/test/scala/system/basic/WskBasicTests.scala

```diff
@@ -814,6 +877,37 @@ class WskBasicTests
         stderr should include regex (s"""Unable to get rule '$name': The requested resource does not exist. \\(code \\d+\\)""")
     }
 
+    it should "return a list of alphabetized rules" in withAssetCleaner(wskprops) {
```

Comment:
you can get fancier and reduce the tests for rules, actions, triggers, and packages into one canonical set.
you'd have to pass in the "type" as in `wsk.trigger` as part of a `Seq(wsk.trigger, wsk.action, ...)` and wrap the test with a foreach on the sequence. I suggest you try that as a later improvement (separate pr) to reduce clone-and-own code.